spec/controllers/event/lists_controller_spec.rb

```diff
@@ -18,29 +18,33 @@
   after   { travel_back }
   subject { assigns(:camps).values.flatten }
 
+  let(:yesterday) { Time.zone.now.to_date - 1.day }
```

Comment:
:musical_note: :notes: 
Yesterday, all my camp-submissions seemed so far away
Now it looks as though they're here to stay
Oh, I believe in yesterday

Suddenly, I realize ActiveSupport has more for me
There's a `Time.zone.now.to_date.yesterday` hanging over me
Oh, yesterday came suddenly

Why `date - 1.day` has to go
I don't know, I will not say
If I said something wrong
Then I long for yesterday
:musical_note: :notes: 

#SCNR 

```suggestion
  let(:yesterday) { Time.zone.now.to_date.yesterday }
```